tests/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Its been removed in https://github.com/joakime/jetty-runtime/commit/dd97cc8f6a051d8a1581eb962a8a4a8a2c639313